content/concepts/asset-manager.md

```diff
@@ -1315,6 +1324,26 @@ There are 2 available parameters for this operation.
 Both parameters are required.
 :::
 
+#### Optimize jpeg
+::: availability versions=Serenity editions=EE
+
+::: info
+Even if you upload a .jpeg image in your asset manager, we automatically convert it into a .png for each operation, in order to avoid quality loss. If you want to convert your images back to .jpeg, you can put this operation last.
```

Comment:
```suggestion
Even if you upload a jpeg image in your asset manager, we automatically convert it into a png for each operation, in order to avoid quality loss. If you want to convert your images back to jpeg, you can put this operation last.
```